Show array value type for `pluginchannels` and `pluginrepos`

antoine2711 opened the pull request:
Since the options pluginchannels and pluginrepos can have arrays of values, this should be what is in the default values.

It's even more obvious with pluginrepos since two consecutives ticks don't render as an empty formatted, but just the normal 2 caracters.

<kbd>

![image](https://github.com/user-attachments/assets/9669e1a0-96f7-492e-9a12-5a2fd7c8e195)

</kbd>


Regards, Antoine

antoine2711 committed:
Show array value type for pluginchannels and pluginrepos

Since the options pluginchannels and pluginrepos can have arrays of values, this should be what is in the default values.

It's even more obvious with pluginrepos since two consecutives ticks don't render as an empty formatted, but just the normal 2 caracters.

niten94 reviewed (approved):
I was asked to merge this if there are no problems but I am not a maintainer. There may be other pull requests that are being focused on right now, but I do not think it would take a while until a maintainer reviews this.

dmaluka commented on runtime/help/options.md:
```diff
@@ -324,11 +324,11 @@ Here are the available options:
 * `pluginrepos`: a list of links to plugin repositories.
 
-    default value: ``
+    default value: `[]`
```
It is not necessarily gonna be immediately obvious to the user what does this mean (even if the user is familiar with json array syntax), so maybe clarify it:

```
    default value: `[]` (empty list)
```

niten94 replied:
@antoine2711:
The only other option with an empty value is `backupdir`, but it also has an explanation like this. The change is small, so would you still be interested to at least add the text only?

antoine2711 replied:
@niten94 : I think it's overkill and obvious, but if you really want it, here it is.

antoine2711 committed:
Merge branch 'micro-editor:master' into patch-1

antoine2711 committed:
Add (empty list) to show that [] is an empty list which might not be obvious to some people

antoine2711 commented:
@JoeKar : what about this PR?

Regards, Antoine

antoine2711 committed:
Update default value description for indent-char

Show that it's an empty string (instead of 2 backticks that are  rendering marks).
